src/styles/blocks/card.scss

```
@@ -0,0 +1,99 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid $cardBorderColor;
+  background: $cardBgColor;
+
+  &__image-container {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__heading {
+    margin: 0 0 4px;
```

```suggestion
    margin-bottom: 4px;
```
rewrite all similar cases